src/themes/teams/components/Text/textVariables.ts

```diff
@@ -1,38 +1,17 @@
-import { ColorValues } from '../../../types'
 import { mapColorsToScheme } from '../../../../lib'
+import { TextVariables } from '../../../base/components/Text/textVariables'
 
-export interface TextVariables {
-  colors: ColorValues<string>
+export interface TeamsTextVariables extends TextVariables {
```

Comment:
Why are not the remaining variables (`atMentions*`, `important*`, `timestamp*`) part of the base theme?
I think that what is in the component API (these are) it should be in base theme as well.